blockingconcurrentqueue.h

```diff
@@ -7,412 +7,16 @@
 #pragma once
 
 #include "concurrentqueue.h"
+#include "lightweightsemaphore.h"
```

Comment:
The README will need updating to indicate that the blocking queue now requires the three headers, not just the two.

Reply:
Done.